frontend/src/routes/layout.css

```
@@ -1,3 +1,39 @@
+}
+
+body {
+  margin: 0;
+  min-height: 100vh;
+  background: linear-gradient(
+    180deg,
+    color-mix(in oklab, var(--ui-color-canvas) 88%, #fff) 0%,
+    var(--ui-color-canvas) 100%
+  );
+  color: var(--ui-color-text);
+  font-family: "Avenir Next", "Manrope", "Segoe UI", sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)